CMakeLists.txt

```diff
@@ -76,7 +76,9 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS 1)
 set (LIBHPDF_STDC_HEADERS 1)
 
 # support all of the different variations of LIBPNG defines in HARU
-set (LIBHPDF_HAVE_LIBPNG ${PNG_FOUND})
+if (${PNG_FOUND})
+set (LIBHPDF_HAVE_LIBPNG TRUE)
```

Comment:
Could you add an indentation of two spaces?

Reply:
At second thought, I think this is nicer:
set (LIBHPDF_HAVE_LIBPNG ${PNG_FOUND})

Reply:
Changed.